src/urllib3/util/response.py

```diff
@@ -60,7 +60,13 @@ def assert_header_parsing(headers):
 
     unparsed_data = None
     if get_payload:  # Platform-specific: Python 3.
```

Comment:
The `# Platform-specific: Python 3` comment was there originally, not added by me; also I note that the docstring for the function says `Only works on Python 3.` (These were modified/added in commit 275cfda998fe55ed10b85fb985f658f795464060 in 2015.) I'm not familiar with the project's usage of this marker and why it's excluded from coverage, so I'll follow your lead.

Reply:
I know you didn't add them, sorry if what I said above was misunderstood :) We use markers like that to exclude a function from test coverage but (imo) a lot of these are actually not great because having more coverage is a good thing! So I try to remove them if we can get 100% test coverage there.

Reply:
I've made that change; it doesn't seem to have changed the coverage.